common/beerocks/bwl/shared/nl80211_client_impl.cpp

```diff
@@ -335,6 +335,13 @@ bool nl80211_client_impl::get_radio_info(const std::string &interface_name, radi
                         channel.is_dfs = true;
                     }
 
+                    // NL80211_FREQUENCY_ATTR_NO_IBSS and NL80211_FREQUENCY_ATTR_NO_IR are set to
+                    // NO_IR. for legacy, check __NL80211_FREQUENCY_ATTR_NO_IBSS
```

Comment:
If its legacy then  why not 
```
                    if (tb_freq[NL80211_FREQUENCY_ATTR_NO_IR] ||
                        tb_freq[__NL80211_FREQUENCY_ATTR_NO_IBSS]) {
```

Reply:
That's fine as long as you don't want to distinguish older kernels and want to treat older NO_IBSS meaning as the new NO_IR.
This solution however does not consider NL80211_FREQUENCY_ATTR_PASSIVE_SCAN. I don't mean we should do it. I don't even know what all these flags mean.